### PR TITLE
Strapi V3 compatibility

### PR DESCRIPTION
**Pull request by roschaefer**
@veitbjarsch no problemo and I'm glad that you keep this package maintained. As requested in #9 , I'm creating this PR which includes changes by @Saunved. Please keep an eye on his/her fork for any updates, e.g. I created this PR there already: https://github.com/Saunved/strapi-provider-email-nodemailer/pull/1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "strapi-provider-email-nodemailer",
-  "version": "2.0.0",
-  "description": "Nodemailer provider for Strapi with attacments support for latest Strapi versions and NTLM auth support",
+  "name": "strapi-provider-email-nodemailer-v3",
```
Comment (roschaefer) [inline package.json:2]:
```suggestion
  "name": "strapi-provider-email-nodemailer",
```
@yutikom are you going to release a new major version? The `-v3` prefix would obviously need to be removed.

File: README.md [hunk 2]
```diff
@@ -1,51 +1,75 @@
-# strapi-provider-email-nodemailer
-
-Nodemailer provider for Strapi with attachments support for latest Strapi versions.
-
-Made by [Automat-IT](https://www.automat-it.com/)
+# strapi-provider-email-nodemailer-v3
+Forked from *strapi-provider-email-nodemailer*
 
+Should work with Strapi 3.0.0+
```
Comment (veitbjarsch) [inline README.md:4]: We should specifiy here that version 3 will work with Strapi 3.x.x.

File: README.md [hunk 3]
```diff
@@ -1,51 +1,75 @@
...
+      username: env('SMTP_USERNAME'),
+      password: env('SMTP_PASSWORD'),
+      // ... any custom nodemailer options
+    },
+    settings: {
+      defaultFrom: 'hello@example.com',
+      defaultReplyTo: 'hello@example.com',
+    }
+  },
+});
```
Comment (veitbjarsch) [inline README.md:31]: I did not checked the current changes in detail. Is there any possibility to provide the user some feedback if the mail configuration is missing or incomplete?

**Comment by Saunved** [comment]
I've merged the PR on my fork @roschaefer @veitbjarsch @yutikom

**Comment by roschaefer** [comment]
@veitbjarsch this was just a PR from a branch of my fork of a fork.

@Saunved has just merged https://github.com/Saunved/strapi-provider-email-nodemailer/pull/1#event-3549135654 :tada: 

So, if you want, we can close this PR and @Saunved wil send a PR from https://github.com/Saunved. That PR shouldn't have the links updated then.

File: README.md (hunk 3 above)
Comment (roschaefer) [reply README.md:31]: I tested this locally using a fake mailserver for development. https://github.com/maildev/maildev

When I debugged the code, I could see that a param `defaultFrom` was provided by Strapi itself. So I decided to rename the other snake_cased param to `defaultReplyTo`.


I haven't tested NTLM at all.

**Comment by Saunved** [comment]
> So, if you want, we can close this PR and @Saunved wil send a PR from https://github.com/Saunved. That PR shouldn't have the links updated then.

I have opened a new PR here: https://github.com/yutikom/strapi-provider-email-nodemailer/pull/11
I have taken the comments on this PR into consideration and made changes wherever applicable. Please review :)

**Comment by roschaefer** [comment]
@Saunved excellent :raised_hands: 

..closing

**Pull request closed without merging**